packages/gollm/gollm_openai/prompts/interventions_from_document.py

```diff
@@ -26,9 +26,10 @@
 3.	`model_id` id a UUID. If the AMR model has an id, you can use it. Otherwise, you can set as the nil UUID "00000000-0000-0000-0000-000000000000".
 4.	For each intervention specified in the policy create an intervention object with the following rules.
     a.	Create a value for `name` from the user-provided text.
-    b.	`appliedTo` should reference the id of the parameter or initial state of the AMR Model. If you cannot find an initial state or parameter that matches the intervention, do not create an intervention object.
-    c.	`type` should be either "state" or "parameter" depending on what the intervention is applied to.
-    d.	create a list of either static or dynamic interventions, but not both.
+    d.	create a list of either static or dynamic interventions, but not both. The other list should be empty.
```

Comment:
Will it matter at all that this goes `a,d` and not `a,b` to the LLM?

Reply:
Who knows... LLM have a mind of their own

Reply:
I added this last line because the LLM was not creating an empty list for the other intervention type. Even though the json schema says it is mandatory